testing/tests/DevExpress.ui/themes.tests.js

```diff
@@ -45,33 +45,15 @@ require("style-compiler-test-server/known-css-files");
     }
 
     function isGoodSelector(selectorText) {
```

Comment:
Previously this was `false`:
```
isGoodSelector(".dx-button .j123-grid")
```

Reply:
It was but it doesn't really make sense for me. I think the main goal for us is not to have any selector that can damage an app outside of our components. Anyway, it's not a big deal to return a class check back. The leading tag is still needed to make some selectors stronger.

Reply:
> It was but it doesn't really make sense for me. 

The test case dates back to 2014. If you and peers are confident in the changes, then fine.

> The leading tag is still needed to make some selectors stronger.

Without an example this need is vague. Rule with the leading `body` look odd to me.